service/src/main/kotlin/fi/espoo/evaka/application/PendingDecisionEmailService.kt

```diff
@@ -137,10 +144,12 @@ WHERE id = :id
     }
 
     private fun getSubject(language: Language): String {
+        val postfix = if (System.getenv("VOLTTI_ENV") == "staging") " [staging]" else ""
```

Comment:
or maybe `val postfix = System.getenv("VOLTTI_ENV").let { if(it == "prod") "" else " [$it]" }`